.gitignore

```diff
@@ -0,0 +1,2 @@
+.history
```

Comment:
Please remove the `.history` and replace it with `/node_modules` 👍🏻

Reply:
I do not have a /node_modules file, I added the history so that all of my history file isn't uploaded on Github